### PR TITLE
FEATURE: Sort queries by last_run_at as default

### PR DESCRIPTION
**Pull request by rishabhnambiar**
lastrun

### CODE REVIEW

File: plugin.rb [hunk 1]
```diff
@@ -938,10 +939,11 @@ def show
 
     def create
       # guardian.ensure_can_create_explorer_query!
-
+      
       query = DataExplorer::Query.from_hash params.require(:query)
-      query.created_at = Time.now.strftime("%b %e, %Y")
+      query.created_at = Time.now
       query.created_by = current_user.id.to_s
+      query.last_run_at = '-'
```
Comment (ZogStriP) [inline plugin.rb]: Feels weird. Why isn't this done in the client-side? This looks like a presentation-only edge case.

Comment (techAPJ) [inline plugin.rb]: I think we should default `last_run_at` to the time query was created instead of keeping it blank.

File: plugin.rb [hunk 2]
```diff
@@ -999,6 +1001,9 @@ def schema
     def run
       check_xhr unless params[:download]
       query = DataExplorer::Query.find(params[:id].to_i)
+      query.last_run_at = Time.now
+      query.save
```
Comment (techAPJ) [inline plugin.rb]: This should be `query.save!`so that we raise an error if any.

File: plugin.rb (hunk 1 above)
Comment (rishabhnambiar) [reply plugin.rb]: Yeah my bad, defaulting this to `Time.now` makes more sense than this. Thanks!!

File: plugin.rb (hunk 2 above)
Comment (rishabhnambiar) [reply plugin.rb]: Doing that returns `NoMethodError - undefined method save! for #<DataExplorer::Query:0x000056235ba9abf8>`. 
I believe this is a built-in function, any ideas?

Comment (techAPJ) [reply plugin.rb]: I see, we have implemented the `save` function in `DataExplorer::Query` manually. Ignore.

Just commit the `last_run_at` change we discussed above and feel free to merge the PR.

Comment (rishabhnambiar) [reply plugin.rb]: Will do, thanks! :)

**Pull request merged**